test/multiverse/lib/multiverse/runner.rb

```diff
@@ -46,6 +46,15 @@ def parse_args(args)
     end
 
     def run(filter = "", opts = {})
+      # This file is generated by TestTimeReporter.
+      # TestTimeReporter cannot be loaded until minitest has been loaded by an
+      # individual suite, so we cannot define this inside the TestTimeReporter
+      # class. Please keep this path and the paths for the reporter constants
+      # the same!
+      # rubocop:disable Lint/NonAtomicFileOperation
+      File.delete(Multiverse::TEST_TIME_REPORT_PATH) if File.exist?(Multiverse::TEST_TIME_REPORT_PATH)
```

Comment:
Satisfying the linter should be easy enough to do here if you bring in FileUtils as you've done elsewhere in the PR:

```diff
+ require 'fileutils'
  ...
- File.delete(Multiverse::TEST_TIME_REPORT_PATH) if File.exist?(Multiverse::TEST_TIME_REPORT_PATH)
+ FileUtils.rm_f(Multiverse::TEST_TIME_REPORT_PATH)
```

Reply:
The line of code you suggest is what I had here originally, but an error was resolved by removing it (though which error, I can't quite remember). I'll try reverting that change and see if things still work.

Reply:
It worked fine this time around! 🙃 
Resolved in: d95ad43f2acd27a98338d0fa5ea8ac32a41d445e